### PR TITLE
Update Firefox CSS support

### PR DESCRIPTION
**Pull request by Elchi3**
Update more Firefox compat data to get closer to 100% real values. How I came to which versions is outlined in this table:

| id(s) | change / explanation |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| css.at-rules.supports | close the range for the preffed version |
| css.properties.align-items.flex_context | Other flex_context have this as 20 as well, seems only consistent. |
| css.properties.align-items.grid_context.start_end | I believe this landed with the initial support for align-items in grid_context |
| css.properties.background-image.element | taken from https://developer.mozilla.org/en-US/docs/Web/CSS/element#Browser_compatibility |
| css.properties.background-image.image-rect | taken from https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-image-rect#Browser_compatibility |
| css.properties.border-inline-end-color css.properties.border-inline-end-style css.properties.border-inline-end-width css.properties.border-inline-start-color css.properties.border-inline-start-style | https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/releases/3/CSS_improvements mentions "-moz-border-*-start and -moz-border-*-end CSS properties are implemented (https://bugzilla.mozilla.org/show_bug.cgi?id=74880)" |
| css.properties.border-radius.4_values_for_4_corners css.properties.border-radius.elliptical_borders | See older revision that this was supported early, so setting "4" when the unprefixed variant became available. https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius$revision/36202 |
| css.properties.box-align (-moz- prefix) | Added in https://bugzilla.mozilla.org/show_bug.cgi?id=71471 |
| css.properties.box-pack (-moz- prefix) | https://bugzilla.mozilla.org/show_bug.cgi?id=71471 |
| css.properties.box-direction (-moz- prefix) | Added in https://bugzilla.mozilla.org/show_bug.cgi?id=93177 |
| css.properties.box-flex (-moz- prefix) | Added in https://bugzilla.mozilla.org/show_bug.cgi?id=70860 |
| css.properties.box-orient (-moz- prefix) | Added in https://bugzilla.mozilla.org/show_bug.cgi?id=70809 |
| css.properties.box-ordinal-group (-moz- prefix) | Added in https://bugzilla.mozilla.org/show_bug.cgi?id=93519 |
| css.properties.box-decoration-break (-moz-background-inline-policy): | was previously documented here and says version 1: https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-background-inline-policy$revision/933439 |
| css.properties.column-rule-style (-moz- prefix) | column-rule-color and column-rule-width also have 3.5 for the -moz- prefix, so this is only consistent. |
| css.properties.display.inline-flex | close the range for the preffed version |
| css.properties.display.xul_box_values css.properties.display.xul_grid_values css.properties.display.xul_stack_values css.properties.display.xul_deck_values css.properties.display.xul_popup_values | Couldn't find an implementation bugs, but other bug reports that appeared with very early Firefoxes, so it must have been in 1.0 or earlier https://bugzilla.mozilla.org/show_bug.cgi?id=271227 |
| css.properties.resize css.properties.resize (-moz- prefix) | https://bugzilla.mozilla.org/show_bug.cgi?id=442228 implemented this prefixed https://bugzilla.mozilla.org/show_bug.cgi?id=553576 implemented this unprefixed Both were in the cycle for Firefox 4, so the prefixed version hasn't actually seen any released product. |
| css.properties.text-combine-upright | close the range for the flag change |

### CODE REVIEW

File: css/at-rules/supports.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
               },
               {
                 "version_added": "17",
-                "version_removed": true,
+                "version_removed": "22",
```
Comment (vinyldarkscratch) [inline css/at-rules/supports.json]: Perhaps we should also add this change to Firefox Android?

File: css/properties/column-rule-style.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
               },
               {
                 "prefix": "-moz-",
-                "version_added": true
+                "version_added": "3.5"
```
Comment (vinyldarkscratch) [inline css/properties/column-rule-style.json]: Same here!

File: css/properties/display.json [hunk 3]
```diff
@@ -640,7 +640,7 @@
                 },
                 {
                   "version_added": "18",
-                  "version_removed": true,
+                  "version_removed": "20",
```
Comment (vinyldarkscratch) [inline css/properties/display.json]: Also here.

File: css/properties/text-combine-upright.json [hunk 4]
```diff
@@ -44,7 +44,7 @@
               },
               {
                 "version_added": "31",
-                "version_removed": true,
+                "version_removed": "41",
```
Comment (vinyldarkscratch) [inline css/properties/text-combine-upright.json]: Finally, here.

**Comment by Elchi3** [comment]
Good point! I mirrored things over to FxA. Thanks for the quick review! :)

**Pull request merged**